src/inmanta/ast/statements/assign.py

```diff
@@ -495,6 +495,8 @@ class ShortIndexLookup(IndexLookup):
     host = ExtendedHost(...)
     resource = std::testing::NullResource(name='test', host=host, ...)
 
+    index std::testing::NullResource(name='test', host=host)
```

Comment:
this is syntactically incorrect, right?

Reply:
I don't know. I asked your help when witting this and you told me it was fine. I don't know how to write this example so that it remains as close to what is was before. What should change?

Reply:
```suggestion
    index std::testing::NullResource(host, name)
```